test/tests/functional/pbs_reservations.py

```diff
@@ -1931,10 +1920,12 @@ def test_delete_idle_resv_job_delete(self):
         now = int(time.time())
         start = now + 30
         idle_timer = 15
-        a = {'reserve_start': start, 'reserve_end': now + 3600,
-             'delete_idle_time': 15}
-        r = Reservation(attrs=a)
-        rid = self.server.submit(r)
+        extra = {'delete_idle_time': idle_timer}
+        rid = self.submit_reservation(user=TEST_USER,
+                                      select='1:ncpus=1',
+                                      start=start,
+                                      end=now + 3600,
+                                      extra_attrs=extra)
```

Comment:
Why submit as TEST_USER, it is the default.  Unless you are checking for it to be TEST_USER, there is no need.  Here and other places.

Reply:
This is most likely because of copy/paste. Since there is no harm in doing this, I'd like to leave it as is.